AUTHORS

```diff
@@ -868,5 +868,125 @@ Super Zheng <superzheng@tencent.com>
 Rafał Pocztarski <r.pocztarski@gmail.com>
 Michael Ruddy <mruddybtc@gmail.com>
 Andy Bettisworth <andy.bettisworth@accreu.com>
+Jörg Krause <joerg.krause@embedded.rocks>
+Alexander Martin <alex@suitupalex.com>
+Prince J Wesley <princejohnwesley@gmail.com>
+janriemer <janriemer@users.noreply.github.com>
+Arthur Gautier <superbaloo+registrations.github@superbaloo.net>
+Martin von Gagern <Martin.vGagern@gmx.net>
+Hideki Yamamura <hideki.yamamura@gmail.com>
+Lenny Markus <lmarkus@paypal.com>
+Nelson Pecora <nelson@yoshokatana.com>
+Xotic750 <xotic750@gmail>
+Nicholas Young <nicholas@nicholaswyoung.com>
+chrisjohn404 <chris.s.johnson4@gmail.com>
+Bo Borgerson <gigabo@gmail.com>
+Mudit Ameta <zeusdeux@gmail.com>
+Adri Van Houdt <adri.van.houdt@gmail.com>
+Max Barinov <me@maxbarinov.com>
+Damian Schenkelman <damian.schenkelman@gmail.com>
+iamchenxin <iamchenxin@gmail.com>
+Nik Nyby <nnyby@columbia.edu>
+Ben Ripkens <bripkens.dev@gmail.com>
+Ryan Sobol <contact@ryansobol.com>
+jona <jona@jona.io>
+Martin Bark <martin@barkynet.com>
+José F. Romaniello <jfromaniello@gmail.com>
+Richard Sun <rsun@media.ucla.edu>
+Sequoia McDowell <sequoia.mcdowell@gmail.com>
+zoubin <zoubin04@gmail.com>
+Robert Jefe Lindstaedt <robert.lindstaedt@gmail.com>
+Saquib <saquibofficial@gmail.com>
+Daniel Sellers <daniel@designfrontier.net>
+Julien Waechter <julien.waechter@gmail.com>
+Vitor Cortez <vitoracortez+github@gmail.com>
+Michael Theriot <michael.lee.theriot@gmail.com>
+Peter Geiss <peter.w.geiss@gmail.com>
+Richard Lau <riclau@uk.ibm.com>
+José Moreira <jose.moreira@findhit.com>
+Tom Atkinson <atkinson.tommy@nsoft.ba>
+Glen Arrowsmith <glen.arrowsmith@gmail.com>
+Jeff Harris <techjeffharris@gmail.com>
+Alexander Makarenko <estliberitas@gmail.com>
+Drew Folta <drew@folta.net>
+Myles Borins <myles.borins@gmail.com>
+Jimb Esser <wasteland@gmail.com>
+Dmitriy Lazarev <wKich@users.noreply.github.com>
+Adam Langley <agl@google.com>
+Kári Tristan Helgason <kthelgason@gmail.com>
+Manuel Valls <manolo@vlrz.es>
+Prayag Verma <prayag.verma@gmail.com>
+Gibson Fahnestock <gib@uk.ibm.com>
+Alan Cohen <alan@ripple.com>
+Chris911 <christophe.naud.dulude@gmail.com>
```

Comment:
"Christophe Naud-Dulude" is right. However my only contribution so far was comments update.